.github/workflows/Pre_release.yml

```
@@ -1,6 +1,14 @@
 name: "Publish Pre-Release"
 on:
   workflow_dispatch:
+    inputs:
+      commit:  
+        # Note: The tag itself will not be based off this commit. As we must first commit the 
+        # updates to the Artifacts.toml, then the tag will be based off that commit.
+        description: 'The commit to branch from when making the tag'
```

YAML requires very little quoting. If you do want to quote this stick to double-quotes. Single quotes are only required for GHA conditionals
```suggestion
        description: The commit to branch from when making the tag
```